nb/updatecenters/extras/nbjavac.impl/build/no-license.txt

```diff
@@ -0,0 +1 @@
+[NO LICENSE SPECIFIED]
```

Comment:
This confirms, that this reintroduces changes, that were already reviewed and in this case is broken. Please drop all changes relating to the nbjavac changes from this and rebase the remaining parts onto master. That should ensure, that no regressions are pulled in.

Reply:
Done by 276330e